astropy/utils/tests/test_data.py

```diff
@@ -2138,10 +2139,10 @@ def test_clear_download_cache_variants(temp_cache, valid_urls):
     assert not is_url_in_cache(u)
 
 
-@pytest.mark.skipif("CI", reason="Flaky on CI")
+@pytest.mark.skipif("CI_WORKFLOW", reason="Flaky/too much external traffic for CI")
```

Comment:
This is enough, right? You wouldn't set this var or CI locally anyway.
```suggestion
@pytest.mark.skipif(os.environ.get("IS_CRON", "false") == "false", reason="Flaky/too much external traffic for CI")
```

Reply:
Yes, should have seen I could set this similarly to the Windows skips, but got a bit confused about the `"CI"` passed as string – why was that anyway.
I'd nonetheless combine both however, so this would still run in interactive tests.